programs/client/Client.cpp

```diff
@@ -1031,19 +1031,30 @@ class Client : public Poco::Util::Application
         if (server_exception)
         {
             bool print_stack_trace = config().getBool("stacktrace", false);
-            std::cerr << "Received exception from server (version " << server_version << "):" << std::endl
-                << getExceptionMessage(*server_exception, print_stack_trace, true) << std::endl;
+            fmt::print(stderr, "Received exception from server (version {}):\n{}\n",
+                server_version,
+                getExceptionMessage(*server_exception, print_stack_trace, true));
             if (is_interactive)
-                std::cerr << std::endl;
+            {
+                fmt::print(stderr, "\n");
+            }
+            else
+            {
+                fmt::print(stderr, "(query: {})\n", full_query);
```

Comment:
May be it will be better to print full query under an option? because what if it is some huge multiquery?

Reply:
The query was printed for the client error case anyway, plus I'm not sure that adding an option just for this worth it.

NOTE: that it will make CI logs more informative, since right now they do not report the query in case of server exception or test hint mismatch.

Reply:
Ok, agree.